src/Microsoft.VisualStudio.Threading.Analyzers/Properties/AssemblyInfo.cs

```diff
@@ -5,14 +5,14 @@
 *********************************************************/
 
 using System;
-using System.Reflection;
 using System.Resources;
 using System.Runtime.InteropServices;
 
 [assembly: CLSCompliant(false)]
 [assembly: ComVisible(false)]
-[assembly: AssemblyTitle("Microsoft.VisualStudio.Threading.Analyzers")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
-[assembly: AssemblyProduct("Microsoft® Visual Studio® Validation")]
```

Comment:
Because they are redundant (and therefore cause build breaks) with the new build .targets that generate this attributes automatically.